### DISCUSSION TITLE
Why are so many files  under Git in Rstudio?

### DISCUSSION THREAD

**Discussion by emmats** [atsa-es/fish550-2023#26] [Q&A]
I just pulled the class git in Rstudio and it seemed that every single file showed up in my git environment and the boxes were checked next to each one. I'm not sure why this happened our how to undo it. At the very least I need to figure out how to uncheck the 100s of files so that I commit just the one I'm working on.

**Comment by eeholmes** [comment]
Darn it. Dylan had the same problem. It has to do with that futzing I was
doing with the Quarto book.

Do you have files you need to save? If not you can delete the repo on your
computer and re-clone to your computer. That's probably the fastest.

On Thu, Apr 27, 2023 at 12:38 PM emmats ***@***.***> wrote:

> I just pulled the class git in Rstudio and it seemed that every single
> file showed up in my git environment and the boxes were checked next to
> each one. I'm not sure why this happened our how to undo it. At the very
> least I need to figure out how to uncheck the 100s of files so that I
> commit just the one I'm working on.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/atsa-es/fish550-2023/discussions/26>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AATNSOVNYHC4ZCRQ4BI23C3XDLDKDANCNFSM6AAAAAAXOJW4LA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
------------------------------------
EE Holmes, NOAA Fisheries, Seattle, WA   206-860-3369 (or 206-419-6164)

**Reply by emmats** [reply]
Eli, I'm just glad it wasn't something I did!
